install.ps1

```diff
@@ -94,7 +103,7 @@ function Test-WebConnection {
 
     $response = $null
     try {
-        $response = Invoke-WebRequest -Uri "https://$url" -UseBasicParsing
+        $response = Invoke-WebRequest -Uri "https://$url" -UseBasicParsing -DisableKeepAlive
```

Comment:
Nice improvement, this function is only used to test the internet connection so we do not need a persistent connection to the server as there are not subsequent requests.